static/configeditor/index.html

```diff
@@ -722,14 +722,14 @@ <h3 id="docker-enable-outbound">Enabling outbound connections (optional)</h3>
 
```

Comment:
Is there still more work to do to blue rinse the Docker install path? Also wondering if the start of the process needs to specify the registry and namespace, e.g. ./run-acd-ce-sh -registry acdcontaineredition.azurecr.io -registry_namespace acd-ce so it makes use of production registry instead of us.icr.io, or is that overridden elsewhere? Expect it was tested so trust it's working as is.

Reply:
There are still references to IBM AppID for securing HTTP Server and migrating from an EAP instance.  Those products could still be in use.
The run-acd-ce.sh script defaults to the production registry and namespace but will take others via parameter in the event additional supported container registries are added.